url/url-setters-a-area.window.js

```diff
@@ -15,15 +21,16 @@ function runURLSettersTests(all_test_cases) {
       if ("comment" in test_case) {
         name += " " + test_case.comment;
       }
-      test(function() {
+      var key = test_case.href.split(":")[0];
```

Comment:
@annevk said elsewhere he'd prefer:

```suggestion
      const key = test_case.href.split(":")[0];
```